### PR TITLE
461 - Fix building room pages being cut-off on mobile views

### PR DESCRIPTION
**Pull request by marktran2**
### Summary of changes
- previously had the mobile view of a room page cut off:
image

- now has been fixed so that after the small mobile breakpoint is crossed, the room page will take up the whole page width instead of having it fixed to 400px
image

### Viewing changes
- run branch locally and have the app in a mobile view (< 600px wide) and click on any building to see card

### CODE REVIEW

File: frontend/views/BuildingDrawer.tsx [hunk 1]
```diff
@@ -95,12 +95,15 @@ const RoomBoxSubheading = styled(Typography)<TypographyProps>(({ theme }) => ({
...
 export const drawerWidth = 400;
+const drawerWidthMobile = "100%";
 
 const BuildingDrawer: React.FC<{ open: boolean }> = ({ open }) => {
   const dispatch = useDispatch();
   const datetime = useSelector(selectDatetime);
   const building = useSelector(selectCurrentBuilding);
   const { status: rooms } = useBuildingStatus(building?.id ?? "");
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
```
Comment (marktran2) [inline frontend/views/BuildingDrawer.tsx:106]: we can keep the building drawer as at most 400px at all times or, like in this case, have it 100% width after mobile breakpoint (if we keep it at most 400px then a new breakpoint will be needed)

**Review by marktran2 — COMMENTED** [review]
self review

**Comment by richard-xue-vrx** [comment]
This looks good to me! On a completely unrelated note, there's a slight issue with the time box not taking up the full space when expanding from 400px to 600px width, but that's something for another time/a matter of opinion on what looks good.

**Review by richard-xue-vrx — APPROVED** [review]
LGTM!

**Comment by JessicaF** [comment]
@marktran2 Could you please add some tests for the BuildingDrawer file? Can just be simple tests checking that some expected text shows up with some basic mocked data

Otherwise the functionality looks good to me!

**Comment by marktran2** [comment]
@JessicaF added tests just recently. Please lmk what u think!

**Review by JessicaF — APPROVED** [review]
Thank you for the tests! Looks good 😎

**Pull request merged**